lib/api/sibling-nav.js

```diff
@@ -4,22 +4,23 @@ import { queryAPI } from "@/lib/fetch";
 export async function addSiblings(entryData, site = "default") {
   if (!entryData) return null;
 
-  const { uri, level } = entryData;
-
-  const siblings = await getSiblings(uri, level, site);
-
+  const { uri, level, parent } = entryData;
+  const parentId = parent?.id;
```

Comment:
conditionally pull out the parent ID (conventionally we use URI as entry identifier but we have to use page ID here because that's what the graphQL filter accepts